lightning/src/routing/router.rs

```diff
@@ -1845,6 +1847,10 @@ mod tests {
 		}
 	}
 
+	fn default_onion_format_flags() -> Vec<u8> {
+		vec![0, 2]
+	}
```

Comment:
Could drop this in favor of using the previously suggested method and calling `le_flags` on the result.